vendor/engines/split_accounts/spec/factories/split_accounts.rb

```diff
@@ -0,0 +1,45 @@
+FactoryGirl.define do
+  factory :split_account do
```

Comment:
I was thinking it would be a good idea to create a `account` factory that is the parent of `nufs_account` and `split_account`.  But I figured it was outside the scope of this PR.